changelog.dd

```diff
@@ -23,6 +23,7 @@ $(BUGSTITLE Language Changes,
 $(BUGSTITLE Compiler Changes,
     $(LI $(RELATIVE_LINK2 deferred_alias, Analysis for aliases in imported modules is deferred.))
     $(LI $(RELATIVE_LINK2 native_tls_osx, Native TLS on OS X 64 bit.))
+    $(LI $(RELATIVE_LINK2 __FILE_FULL_PATH__, Special trait replaced by the source file's absolute file name.)
```

Comment:
Actually I just googled it.  It does look at behave like a manifest constant.  However, it isn't defined in the same way (must be defined by the compiler vs somewhere in the application's code base).  In the current compiler implementation, it required a new token be added, which makes it seem like a keyword.  What do you think?